tests/test_models/test_forward/test_sot_forward.py

```diff
@@ -141,14 +141,23 @@ def test_stark_forward():
 @pytest.mark.parametrize('cfg_file', [
     'sot/siamese_rpn/siamese_rpn_r50_20e_lasot.py',
     'sot/siamese_rpn/siamese_rpn_r50_20e_vot2018.py',
-    'sot/stark/stark_st2_r50_50e_got10k.py'
+    'sot/stark/stark_st2_r50_50e_got10k.py',
+    'sot/mixformer/mixformer_cvt_500e_got10k.py'
 ])
 def test_sot_test_forward(cfg_file):
     config = _get_config_module(cfg_file)
     model = copy.deepcopy(config.model)
     sot = build_model(model)
     sot.eval()
 
+    device = torch.device('cpu')
+    if config.model.type == 'MixFormer':
+        if not torch.cuda.is_available():
+            return
+        else:
+            device = torch.device('cuda')
+    sot = sot.to(device)
```

Comment:
```suggestion
    device = torch.device('cpu')
    if torch.cuda.is_available():
        device = torch.device('cuda')
    sot = sot.to(device)
```
It seems that we skip the ut of MixFormer when `torch.cuda.is_available() is False` by `return`

Reply:
We write this because the `score decoder head` of mixformer uses `mmcv.ops.prroi_pool.PrRoIPool`, which doesn't support CPU. This change will lead to exception when using cpu device.

Reply:
We write this because the `score decoder head` of mixformer uses `mmcv.ops.prroi_pool.PrRoIPool`, which doesn't support CPU. This change will lead to exception when using cpu device.